src/components/icon/icon.tsx

```diff
@@ -105,6 +105,8 @@ export class Icon {
             this.svg = '';
             this.slError.emit({ status: file.status });
           }
+        } else {
+          this.slError.emit({ status: file.status });
```

Comment:
I decided not to reset the svg in this case because it's also not done in the case of a network error (catch below).